pages/doc/service_and_applications_view.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Services and Applications Page (Controlled Release)
+keywords: data, distributed tracing
+tags: [tracing]
+sidebar: doc_sidebar
+permalink: service_and_application_view.html
+summary: Overview of services and applications that send data to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront).
+---
+
+{% include important.html content="This feature is enabled only for selected customers. To participate, contact your account representative or [technical support](wavefront_support_feedback.html#support)."%}
+
+As part of troubleshooting an application, you need an overview of the services and applications that send data to Operations for Applications. Use the table view or grid view to understand the health of each service, and troubleshoot when your applications or services run into issues.
+
+{% include note.html content="If you are using eBPF, you don't send traces to our service. Therefore, you don't see the option to drill down to the Traces Browser." %}
+
+<a name="table_view"></a>
+
+## Table View Features
+
+View the list of applications and services. You can see the Request, Error, and Duration (RED) metrics at a glance and sort the data.
+
+To load the table view:
+1. Log in to your service instance.
+1. From the toolbar, select **Applications** > **Application Status** and click the **Table View** icon ( <img src="images/tracing_appmap_table_view_icon.png"
+style="vertical-align:text-bottom;width:28px" alt="icon to click to get the table view"/> )
+
+![the image shows the table view of all the applications that send data to Operations for Applications. It has helpers to show you what to do with each UI section. For example, how to filter applications or services, change the table settings or the legend settings, and how to change back to the application map view or the grid view](images/application_monitoring_table_view.png)
+
+Using the table view, you can:
+* Examine the applications and services, or search for a particular application or service by applying filters.
+  <br/>You can refine your search further by applying one or more filters, such as the cluster, shard, or span.kind.
+* Click the name of a service to drill down to the Service Dashboard.
+* In the View column, click **Service Map** to see how a specific service communicates with the other services in the application.
+* Sort data:
+  - Sort by application and service names.
+  - Sort by cluster names.
+* Create smart alerts that filter noise and capture true anomalies by clicking the ellipsis icon and selecting **Create Alert**. See [Create an alert](#create-alerts).
+* View the alerts that fire for a service by clicking the ellipsis icon and selecting **View Alerts**. The number of alerts that fire for the service are shown in the **Alerts** column.
```

Comment:
This can be fixed later, but probably should be:

The number of alerts that fire for the service are --> The number of alerts that fire for the service is?